packages/react/src/components/Toggletip/__tests__/Toggletip-test.js

```diff
@@ -4,23 +4,183 @@
+        </Toggletip>
+      );
+
+      expect(container.firstChild).toHaveClass('custom-class');
+    });
+    it.each([
+      ['left', 'cds--popover--left'],
+      ['right', 'cds--popover--right'],
+      ['top', 'cds--popover--top'],
+      ['bottom', 'cds--popover--bottom'],
+      ['top-left', 'cds--popover--top-left'],
+      ['top-right', 'cds--popover--top-right'],
+      ['bottom-left', 'cds--popover--bottom-left'],
+      ['bottom-right', 'cds--popover--bottom-right'],
+      ['left-top', 'cds--popover--left-top'],
+      ['left-top', 'cds--popover--left-top'],
+      ['right-top', 'cds--popover--right-top'],
+      ['right-bottom', 'cds--popover--right-bottom'],
+    ])(
+      'should support different alignments with the `align` prop',
+      (align, className) => {
+        const { container } = render(
+          <Toggletip data-testid="toggletip" align={align}>
+            <span>test</span>
+          </Toggletip>
+        );
+
+        expect(container.firstChild).toHaveClass(className);
+      }
+    );
+    it('should initially be open if `defaultOpen` is set to true', () => {
+      const { container } = render(
+        <Toggletip data-testid="toggletip" defaultOpen>
+          <span>test</span>
+        </Toggletip>
+      );
+
+      expect(container.firstChild).toHaveClass('cds--toggletip--open');
+      expect(container.firstChild).toHaveClass('cds--popover--open');
+    });
+
+    it('should close when the browser window loses focus', () => {
+      const { container } = render(
+        <Toggletip data-testid="toggletip" defaultOpen>
+          <span>test</span>
+        </Toggletip>
+      );
+
+      fireEvent.blur(window);
+
+      expect(container.firstChild).not.toHaveClass('cds--toggletip--open');
+      expect(container.firstChild).not.toHaveClass('cds--popover--open');
+    });
+
+    it.todo(
+      'should return to the trigger button if the menu is closed while focus is still inside the menu'
+    );
+
+    it('should not close when the menu itself is clicked', () => {
+      const { container } = render(
+        <Toggletip data-testid="toggletip" defaultOpen>
+          <div data-testid="innerDiv">test</div>
+        </Toggletip>
+      );
+
+      userEvent.click(screen.queryByTestId('innerDiv'));
+
+      expect(container.firstChild).toHaveClass('cds--toggletip--open');
+      expect(container.firstChild).toHaveClass('cds--popover--open');
+    });
+
+    it('should be closed when the focus leaves the tooltip', () => {
+      const { container } = render(
+        <div>
+          <Toggletip data-testid="toggletip" defaultOpen>
+            <span>test</span>
+          </Toggletip>
+
+          <span>focus-here</span>
+        </div>
+      );
+
+      const element = screen.getByText('focus-here');
+
+      fireEvent.focus(element);
+
+      expect(container.firstChild).not.toHaveClass('cds--toggletip--open');
+      expect(container.firstChild).not.toHaveClass('cds--popover--open');
+    });
```

Comment:
I think in this case since it's being wrapped in a `div`, container.firstChild is going to be the div itself so the assertion for it to not have the class "cds--toggletip--open" will always be `true`. This test seems to pass even if you remove the `fireEvent.focus(element);` line